src/meta/src/manager/catalog/fragment.rs

```diff
@@ -281,6 +279,9 @@ where
             let chain_actor_ids = table_fragments.chain_actor_ids();
             let mut dependent_tables = Vec::with_capacity(dependent_table_ids.len());
             for dependent_table_id in dependent_table_ids {
+                if !map.contains_key(&dependent_table_id) {
```

Comment:
In which case the dependent table id itself is not in the `table_fragments`? 👀 Could you please add some comments here?